packages/app/src/cli/services/context/prompts.ts

```diff
@@ -69,31 +71,38 @@ export async function deployConfirmationPrompt(
     return true
   }
 
-  const confirmationMessage = (() => {
-    switch (deploymentMode) {
-      case 'legacy':
-        return 'Yes, deploy to push changes'
-      case 'unified':
-        return 'Yes, release this new version'
-      case 'unified-skip-release':
-        return 'Yes, create this new version'
-    }
-  })()
-
-  return renderConfirmationPrompt({
-    message: question,
-    infoTable,
-    confirmationMessage,
-    cancellationMessage: 'No, cancel',
-  })
+  const promptText = {message: question, infoTable}
+  const appExists = Boolean(appTitle)
+  const isDangerous = appExists && infoTable.some((section) => section.header.match(/Remove/))
```

Comment:
🤔 this check makes sense but I can see it being a bit brittle if decide to change the copy. Could we pass a boolean from outside that more clearly marks the prompt as dangerous?

Reply:
Good feedback! Changed the code to be more stable.